cmd/release-controller/http_changelog.go

```diff
@@ -40,6 +66,30 @@ func (c *Controller) renderChangeLog(w http.ResponseWriter, fromPull string, fro
 			return
 		}
 
+		op, err := c.releaseInfo.ReleaseInfo(toPull)
+		if err != nil {
+			ch <- renderResult{err: fmt.Errorf("could not get release info for pullSpec %s: %v", toPull, err)}
+			return
+		}
+
+		releaseInfo := releaseInfoConfig{}
+		if err := json.Unmarshal([]byte(op), &releaseInfo); err != nil {
+			ch <- renderResult{err: fmt.Errorf("could not unmarshal release info for pullSpec %s: %v", toPull, err)}
+			return
+		}
+
+		// TODO: There is an inconsistency with what is returned from ReleaseInfo (amd64) and what
+		//       needs to be passed into the RHCOS diff engine (x86_64).
+		architecture := ""
+		archExtension := ""
+
+		if releaseInfo.Config.Architecture == "amd64" {
+			architecture = "x86_64"
+		} else {
+			architecture = releaseInfo.Config.Architecture
```

Comment:
so the rhel / go values match for z and power?

Reply:
Correct.  The only discrepancy was with amd64/x86_64.